### DISCUSSION TITLE
Erro ao Tentar Conectar Via php

### DISCUSSION THREAD

**Discussion by valdirdill** [HashLoad/horse#393] [General]
Bom dia...alguma sugestão do porque esse código abaixo não funciona ao requisitar um servidor horse?

_<?
	$url = "http://200.98.128.65:8050/ping";

    $dadosToConexao = array(
 	'http' => array(
    'method'  => 'GET'));
        
  	$streamContext = stream_context_create($dadosToConexao);	
	$json = file_get_contents($url, false, $streamContext);	
?>_

Dá o erro: Warning: file_get_contents(http://200.98.128.65:8050/serverAtivo): failed to open stream: Connection timed out in 

Obs.: essa mesma url acima, se colocar direto na barra de endereço do browser, aí funciona...no postman também funciona tudo certinho...
Pior é que estava funcionando normal. Do nada não funfa mais.
Server hospedado no Uol...

Qualquer dica ajuda, rs. Obrigado

**Comment by viniciussanchez** [comment]
Não faço ideia... mas aqui pra mim deu 404 esse link.

**Reply by valdirdill** [reply]
Deu 404 porque estava mexendo aqui, rs.
Fiz um código/url mais simples para se analisar, e que dá o mesmo problema. Se puder testar...
Se colocar esta url http://200.98.128.65:8050/ping no browser, funciona, mas o código abaixo dá erro.
< ?
$url = "http://200.98.128.65:8050/ping";
$response = file_get_contents($url);
 ? >

Obrigado!

**Reply by valdirdill** [reply]
Ao que parece, o problema está no servidor Uol, que de ontem para hoje, mudou alguma coisa e não deixa mais buscar dados pelo file_get_contents(. 
Fizemos um teste colocando a mesma aplicação em outro cloud e funciona normal, como funcionava no Uol. 
Sinistro.

**Comment by viniciussanchez** [comment]
deve ser alguma configuração lá entao mesmo
